src/pages/gen2/build-a-backend/data/connect-to-existing-database/index.mdx

```diff
@@ -0,0 +1,398 @@
+  schema,
+  rdsSchema
+])
+
+export type Schema = typeof schema
+export const data = defineData({
+  schema: combinedSchema
+})
+```
+
+## Step 4 - Deploy your Data resources using the cloud sandbox
+
+Finally, you can now validate your Data resources with your cloud sandbox:
+
+```bash
+npx amplify sandbox
+```
+
+## Rename generated models and fields
+
+```ts
+import { a, defineData } from '@aws-amplify/backend'
+import { schema as rdsSchema } from './schema.rds.ts'
+
+// Rename models or fields to be more idiomatic for frontend code
+// highlight-start
+rdsSchema.models.todos.renameTo("Todo")
+rdsSchema.models.todos.fields.title.renameTo("content")
+// highlight-end
+
+const schema = a.schema({
+  Todo: a.model({
+    content: a.string()
+    isDone: a.boolean()
+  })
+}).authorization([a.allow.public()])
+
+const combinedSchema = a.schema.combine([
+  schema,
+  rdsSchema
+])
+
+export type Schema = typeof schema
+export const data = defineData({
+  schema: combinedSchema
+})
+```
+
+Step 5 - Configure the `amplify.yml` TK
+
+Customers need to update the `amplify.yml` to generate the DB schema if the DB is in a different VPC. Etc Etc. TK
+
+TK: npx amplify generate config --connection-string $BRANCH_CONNECTION_STRING
+
+## Add relationships between tables
+
+```ts
+import { a, defineData } from '@aws-amplify/backend'
+import { schema as rdsSchema } from './schema.rds.ts'
+
+// highlight-start
+rdsSchema.models.Todo.addRelationships({
+  comments: a.hasMany("Comment").references("todo_id"),
+  notes: a.hasMany("Note").references("todoId")
+})
+// highlight-end
+
+const ddbSchema = a.schema({
+   Note: a.model({
+     content: a.string(),
+     todoId: a.id()
+     todo: a.belongsTo("Todo").references("todoId")
+   })
+})
+
+const combinedSchema = a.schema.combine([rdsSchema, ddbSchema])
+
+export type Schema = typeof combinedSchema
+export const data = defineData({
+  schema: combinedSchema
+})
+```
+
+## Invoke SQL statements with custom queries and mutations
+
+You can either define inline SQL statements or reference a file path containing custom sql statements in designated files. If both inline and custom statements are set, the sandbox/build should error out.
+
+```ts
+
+import { a, defineData } from '@aws-amplify/backend'
+import { schema as rdsSchema } from './schema.rds.ts'
+
+// Add custom mutations or queries that execute SQL statements
+// highlight-start
+rdsSchema.addQueries({
+  searchTodoBasedOnGeolocation: a.query()
+    .arguments({
+      lat: a.float(),
+      long: a.float(),
+      radiusInMeters: a.float()
+    })
+    .returns(a.ref("Todo").array())
+    .handler(a.handler.inlineSql`
+      SELECT *
+      FROM todos
+      WHERE ST_DistanceSphere(
+        location,
+        ST_MakePoint(:lat, :long)
+      ) <= :radiusInMeters`
+    )
+    .authorization([a.allow.public()])
+})
+// highlight-end
+
+// highlight-start
+rdsSchema.setSqlStatementFolderPath('./aurora-statements/'))
```

Comment:
No longer supported. `a.handler.sqlReference('./relativePathToFile.sql')` is how we specify a path for the file containing SQL statements
```suggestion
```